Workflow/Sources/WorkflowNode.swift

```diff
@@ -74,11 +74,11 @@ final class WorkflowNode<WorkflowType: Workflow> {
         onOutput?(output)
     }
 
-    func render() -> WorkflowType.Rendering {
-        WorkflowLogger.logWorkflowStartedRendering(ref: self)
+    func render(isRootNode: Bool = false) -> WorkflowType.Rendering {
```

Comment:
Do we need to also pass this in the render tester as well?

Reply:
i don't think so, since that type doesn't invoke the render method of the internal representation of a node (it calls into the 'proper' render method of the client-specified Workflow).

Reply:
Ah righto